subworkflows/nf-core/bam_rseqc/main.nf

```diff
@@ -129,21 +133,23 @@ workflow BAM_RSEQC {
     tin_txt = Channel.empty()
     if ('tin' in rseqc_modules) {
         RSEQC_TIN ( ch_bam_bai, ch_bed )
-        tin_txt     = RSEQC_TIN.out.txt
+        ch_tin     = RSEQC_TIN.out.txt
         ch_versions = ch_versions.mix(RSEQC_TIN.out.versions.first())
     }
 
     emit:
-    bamstat_txt                     // channel: [ val(meta), txt ]
+    ch_bamstat                     // channel: [ val(meta), txt ]
```

Comment:
I think if you're going to make these ones 'ch_' prefixes, you'll have to change the others too ;-)